build.gradle

```diff
@@ -23,7 +24,7 @@ ext {
     licenseUrl = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
     localGHPagesRepoDir = '../grain.gh-pages'
     localGHPagesRepoCanonicalPath = new File(localGHPagesRepoDir).canonicalPath
-    javaCompatibilityVersion = 1.7
+    javaCompatibilityVersion = 1.8
```

Comment:
Why not? Java 7 is not supported anymore. On the other hand JDK 8 has a lot of very convenient features that are useful even in groovy.
For current code it does not matter though.

P. S. Hey people, Java 9 is coming! Java 7 is 6 years old.